src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2199,6 +2199,49 @@ private function addMailerSection(ArrayNodeDefinition $rootNode, callable $enabl
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('dkim_signer')
+                            ->addDefaultsIfNotSet()
+                            ->fixXmlConfig('option')
+                            ->canBeEnabled()
+                            ->info('DKIM signer configuration')
+                            ->children()
+                                ->scalarNode('key')
+                                    ->info('Key content, or path to key (in PEM format with the `file://` prefix)')
+                                    ->defaultValue('')
+                                    ->cannotBeEmpty()
+                                ->end()
+                                ->scalarNode('domain')->defaultValue('')->end()
+                                ->scalarNode('select')->defaultValue('')->end()
+                                ->scalarNode('passphrase')->info('A passphrase of the private key if any')->end()
+                                ->arrayNode('options')
+                                    ->performNoDeepMerging()
+                                    ->normalizeKeys(false)
+                                    ->useAttributeAsKey('name')
+                                    ->defaultValue([])
```

Comment:
this default value is useless: all prototyped array nodes automatically have a default value anyway.